pkg/services/featuremgmt/registry.go

```diff
@@ -242,5 +242,11 @@ var (
 			Description: "Use dynamic labels instead of alias patterns in CloudWatch datasource",
 			State:       FeatureStateStable,
 		},
+		{
+			Name:            "coremodelValidation",
```

Comment:
would it be more appropriate to flag "validateDashboardSchema" or something like that -- I don't see any reason to have a flag affect the behavior of the coremodel registry, just a toggle to see if we should use it or skip it

Reply:
Yeah i considered a few paths to this, including the narrowest-possible approach of having a feature flag named just for validating dashboards.

I ended up with this based on the thinking that if we accept that:

- We need a schema for more than one kind of object in the short to medium term
- Coremodels can provide that general system (and can be refactored to be thema-agnostic if necessary, as i've noted elsewhere)

Then we end up with a possible need for a two-dimensional matrix of possible things to grant feature flags on: `O` (number of standard schema operations) `C` (number of coremodels). `OxC` is already a lot of potential feature flags - and it's another dimension if we want to differentiate where these operations are run - e.g. on HTTP handlers vs. when pulling objects from disk.

We may end up wanting to grant such control. But by starting with a coarse-grained approach that just gives single-`O`-dimension control, we avoid feature flag detritus in the event that we _don't_ want to grant that control, while still allowing the possibility of adding that control later (we can always add more flags and require multiple to be on). By contrast, if we start with an `OxC` approach, we're almost guaranteed to have to add more.

Reply:
also if i understand the CEL part of feature flag checking, we could actually make it more complex than just a bool, right? if so, one more reason to pick this naming - we get `OxC` within a single flag, because the value in the flag comprises the `C` dimension

Reply:
OK, will change